src/vpn-api/utils/index.ts

```diff
@@ -20,5 +20,5 @@ export { clients };
 export { HAProxy } from './haproxy';
 export { Netmask } from './netmask';
 export { VpnManager } from './openvpn';
-export { pooledRequest } from './request';
+export { request, pooledRequest } from './request';
```

Comment:
?

Reply:
It needs to be exported for the tests… I'm not entirely sure how the previous PR which changed the naming of this variable passed. Although now you point it out I'll just import `request` directly from `utils/request.ts` in the tests…